llvm/docs/SPIRVUsage.rst

```diff
@@ -114,15 +118,17 @@ Example:
 
 ``-target spirv64v1.0`` can be used to compile for SPIR-V version 1.0 with 64-bit pointer width.
 
+``-target spirv64-amd-amdhsa`` can be used to compile for AMDGCN flavoured SPIR-V version 1.6 with 64-bit pointer width
```

Comment:
I'm not sure how the number of the version 1.6 will be deducted from just `spirv64-amd-amdhsa`.

Reply:
Whoops, that was a mistake, fixed.